source/interact-data/modify-results.txt

```diff
@@ -0,0 +1,281 @@
+
+The following code excludes the ``year`` field from returned ``Band``
+objects:
+
+.. literalinclude:: /includes/interact-data/modify-results.rb
+   :start-after: start-without
+   :end-before: end-without
+   :language: ruby
+   :dedent:
+
+.. important:: _id Field
+
+   {+odm+} requires the ``_id`` field for various operations, so you
+   *cannot* exclude the ``_id`` field or the ``id`` alias from results.
+   If you pass ``_id`` or ``id`` to the ``without()`` method, {+odm+}
+   ignores it.
+
+.. _mongoid-data-sort:
+
+Sort Results
+------------
+
+You can sort the order that {+odm+} returns documents by using the
+``order()`` and ``order_by()`` methods.
+
+These methods accept a hash that indicates which fields to order the
+documents by, and whether to use an ascending or descending order for
+each field.
+
+You can specify the sort direction in the following ways:
+
+- Integers ``1`` (ascending) and ``-1`` (descending)
+  - Example: ``Band.order(name: 1, year: -1)``
+
+- Symbols ``:asc`` and ``:desc``
+  - Example: ``Band.order(name: :asc)``
+
+- Strings ``"asc"`` and ``"desc"``
+  - Example: ``Band.order_by(name: "asc", year: "desc")``
+
+The ``order()`` method also accepts the following sort specifications:
+
+- Array of two-element arrays:
+
+  - Strings
+    - Example: ``Band.order([['name', 'asc'], ['year', 'desc']])``
+
+  - Symbols
+    - Example: ``Band.order([[:name, :asc]])``
+
+- ``asc`` and ``desc`` methods on symbols
+  - Example: ``Band.order(:name.asc, :year.desc)``
+
+- SQL syntax
+  - Example: ``Band.order('name desc')``
+
+You can also use the ``asc()`` and ``desc()`` methods instead of using
+``order()``:
+
+.. code-block:: ruby
+
+   Band.asc('name').desc('year')
+
+If you chain sort specifications, the first call defines the most
+significant criteria and the newest call defines the least significant
+one.
+
+.. TODO update link in the following note for scope
+
+.. note:: Sorting in Scopes
+
+   If you define a scope on your model that includes a sort specification,
+   the scope sort takes precedence over the sort specified in a query, as the
+   default scope is evaluated first.
+
+.. _mongoid-data-skip-limit:
+
+Paginate Results
+----------------
+
+{+odm+} provides the ``limit()``, ``skip()``, and ``batch_size()``
+pagination operators that you can use on ``Criteria`` objects. The
+following sections describe how to use these operators.
+
+Limit Number of Results
+~~~~~~~~~~~~~~~~~~~~~~~
+
+You can limit the number of results that {+odm+} returns by using the
+``limit()`` method.
+
+The following code retrieves a maximum of ``5`` documents:
+
+.. literalinclude:: /includes/interact-data/modify-results.rb
+   :start-after: start-limit
+   :end-before: end-limit
+   :language: ruby
+   :dedent:
+
+Skip Results
+~~~~~~~~~~~~
+
+You can skip a specified number of results by using the ``skip()``
+method, or its alias ``offset()``. If you chain a ``limit()`` call, it
+is applied after documents are skipped. 
```

Comment:
S: might be useful to have an example or admonition for this, actually! probably a common question.

```suggestion
method, or its alias ``offset()``. If you chain a ``limit()`` call
to the ``skip()`` method, the limit
is applied after documents are skipped. 
```